introduction.md

```diff
@@ -1,2 +1,12 @@
 Hello, my name is Ryan Magoon and I am the founder of the forked dojo.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?